lib/xandra/cluster/pool.ex

```diff
@@ -327,8 +317,7 @@ defmodule Xandra.Cluster.Pool do
       )
       when is_peername(peername) do
     # Not connected anymore, but we're not really sure if the whole host is down.
-    data = put_in(data.peers[peername].status, :up)
-    data = stop_pool(data, data.peers[peername].host)
```

Comment:
Wait, why were doing this and why are we not doing it anymore?

Reply:
Good point, we should also `maybe_start_pools` after `stop_pool`.